client/my-sites/checklist/main.jsx

```diff
@@ -0,0 +1,183 @@
+import EmptyContent from 'components/empty-content';
+import FormattedHeader from 'components/formatted-header';
+import isSiteAutomatedTransfer from 'state/selectors/is-site-automated-transfer';
+import Main from 'components/main';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import QuerySiteChecklist from 'components/data/query-site-checklist';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import { createNotice } from 'state/notices/actions';
+import { getCurrentUser } from 'state/current-user/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { isJetpackSite, getSiteSlug } from 'state/sites/selectors';
+import { recordTracksEvent } from 'state/analytics/actions';
+import { requestGuidedTour } from 'state/ui/guided-tours/actions';
+import { requestSiteChecklistTaskUpdate } from 'state/checklist/actions';
+
+class ChecklistMain extends PureComponent {
+	getHeaderTitle( displayMode ) {
+		const { translate } = this.props;
+
+		if ( displayMode === 'free' ) {
+			return translate( 'Your site has been created!' );
+		}
+
+		return translate( 'Thank you for your purchase!' );
+	}
+
+	getSubHeaderText( displayMode ) {
+		const { translate } = this.props;
+
+		if ( displayMode === 'gsuite' ) {
+			return translate(
+				'We emailed %(email)s with instructions to complete your G Suite setup. ' +
+					'In the mean time, let’s get your new site ready for you to share. ' +
+					'We’ve prepared a list of things that will help you get there quickly.',
+				{
+					args: {
+						email: this.props.user.email,
+					},
+				}
+			);
+		}
+
+		return translate(
+			"Now that your site has been created, it's time to get it ready for you to share. " +
+				"We've prepared a list of things that will help you get there quickly."
+		);
+	}
+
+	renderHeader( completed, displayMode ) {
+		const { translate } = this.props;
+
+		if ( completed ) {
+			return (
+				<Fragment>
+					<img
+						src="/calypso/images/signup/confetti.svg"
+						aria-hidden="true"
+						className="checklist__confetti"
+						alt=""
+					/>
+					<FormattedHeader
+						headerText={ translate( 'Congratulations!' ) }
+						subHeaderText={ translate(
+							"You have completed all your tasks. Now let's tell people about it. Share your site."
+						) }
+					/>
+					<ChecklistShowShare
+						className="checklist__share"
+						siteSlug={ this.props.siteSlug }
+						recordTracksEvent={ this.props.track }
+					/>
+				</Fragment>
+			);
+		}
+
+		if ( displayMode ) {
+			return (
+				<Fragment>
+					<img
+						src="/calypso/images/signup/confetti.svg"
+						aria-hidden="true"
+						className="checklist__confetti"
+						alt=""
+					/>
+					<FormattedHeader
+						headerText={ this.getHeaderTitle( displayMode ) }
+						subHeaderText={ this.getSubHeaderText( displayMode ) }
+					/>
+				</Fragment>
+			);
+		}
+
+		return (
+			<FormattedHeader
+				headerText={ translate( 'Welcome back!' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Welcome Back :)' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/checklist-to-my-plan-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).